src/SFA.DAS.ApprenticeApp.Pwa/Controllers/TasksController.cs

```diff
@@ -40,7 +40,7 @@ public async Task<PartialViewResult> ToDoTasks()
             if (!string.IsNullOrEmpty(apprenticeId))
             {
                 var apprenticeshipId = Claims.GetClaim(HttpContext, Constants.ApprenticeshipIdClaimKey);
-                var taskResult = await _client.GetApprenticeTasks(long.Parse(apprenticeshipId), Constants.ToDoStatus, new DateTime(DateTime.Now.Year, 1, 1), new DateTime(DateTime.Now.Year, 12, 12));
+                var taskResult = await _client.GetApprenticeTasks(long.Parse(apprenticeshipId), Constants.ToDoStatus, new DateTime(DateTime.Now.Year, 1, 1), new DateTime(DateTime.Now.Year, 12, 31));
```

Comment:
Please lets keep this 12,31 now. We will effectively remove this and replace with the same once we integrate the year switcher